src/content/docs/apm/apm-ui-pages/monitoring/view-slow-query-details.mdx

```diff
@@ -10,19 +10,17 @@ metaDescription: 'APM: How to view slow query data, and how to enable and config
 redirects:
   - /docs/apm/applications-menu/monitoring/viewing-slow-query-details
   - /docs/apm/applications-menu/monitoring/view-slow-query-details
-freshnessValidatedDate: never
+freshnessValidatedDate: 2024-06-07 
 ---
 
-In APM, when [transaction traces](/docs/apm/transactions/transaction-traces/introduction-transaction-traces) are collected, there may be additional <DoNotTranslate>**Slow query**</DoNotTranslate> data available on the [<DoNotTranslate>**Databases**</DoNotTranslate>](/docs/apm/applications-menu/monitoring/databases-slow-queries-page) page. Let's look at what slow query data is, where to find it, and how to configure it.
 
-You can also use histograms and heatmaps to visualize the distribution of values for different metrics. This is useful to understand slow page performance. To learn how, watch this short video (4:18 minutes).
+import apmApmDatabasesUi from 'images/apm_screenshot-full_apm-databases-ui.webp'
+
 
-<Video
-  id="Gh4Km9aJoBE"
-  type="youtube"
-/>
 
-## Slow query data samples [#about]
+In APM, when [transaction traces](/docs/apm/transactions/transaction-traces/introduction-transaction-traces) are collected, there may be additional <DoNotTranslate>**Slow query**</DoNotTranslate> data available on the [<DoNotTranslate>**Databases**</DoNotTranslate>](/docs/apm/applications-menu/monitoring/databases-slow-queries-page) page. Let's look at what slow query data is, where to find it, and how to configure it.
+
+## Sampling of slow query data [#about]
 
 When [transaction traces](/docs/apm/transactions/transaction-traces/introduction-transaction-traces) are reported, the slow queries feature samples the slowest database queries in those traces and reports data about them on the [<DoNotTranslate>**Databases**</DoNotTranslate>](/docs/apm/applications-menu/monitoring/databases-slow-queries-page) page. Slow query data can include:
```

Comment:
This is also a passive sentence:  `When [transaction traces](/docs/apm/transactions/transaction-traces/introduction-transaction-traces) are reported,`

Reply:
@nbaenam I think passive voice might work here because the subject who is doing the reporting might distract from the secondary subject (the slow queries feature that does the sampling). I'm opting to keep the passive in this case.